src/core/ConfigManager.cpp

```diff
@@ -369,15 +369,14 @@ void ConfigManager::loadConfigFile()
 		m_flDir = ensureTrailingSlash( QDir::home().absolutePath() );
 	}
 
-	if( m_ladDir.isEmpty() || m_ladDir == QDir::separator() ||
-			( !m_ladDir.contains( ':' ) && !QDir( m_ladDir ).exists() ) )
+	if( m_ladDir.isEmpty()  )
 	{
 #if defined(LMMS_BUILD_WIN32)
 		m_ladDir = qApp->applicationDirPath() + "/plugins/ladspa" + QDir::separator();
 #elif defined(LMMS_BUILD_APPLE)
 		m_ladDir = qApp->applicationDirPath() + "/../lib/lmms/ladspa/";
 #else
-		m_ladDir = qApp->applicationDirPath() + '/' + LIB_DIR + "/ladspa/";
+		m_ladDir = qApp->applicationDirPath() + '/' + LIB_DIR + "lmms/ladspa/";
```

Comment:
FWIW, all major OSes support double (or triple, etc) -slashes in paths. There's also [QDir::cleanPath](http://doc.qt.io/qt-4.8/qdir.html#cleanPath) to strip redundant "/", "." and ".." if desired for formatting purposes.

It may be worth concatenating with an extra "/" for consistency or just in case LIB_DIR is ever modified to not have a trailing slash and then cleaning it with `cleanPath` afterward. This would also fix the display on apple caused by using ".." in `qApp->applicationDirPath() + "/../lib/lmms/ladspa/"` (e.g. /path/to/bin/../lib/lmms/ladspa would become /path/to/lib/lmms/ladspa)